src/SignalR/common/testassets/Tests.Utils/InProcessTestServer.cs

```diff
@@ -92,6 +92,10 @@ private async Task StartServerInner()
             var url = "http://127.0.0.1:0";
 
             _host = new HostBuilder()
+                .ConfigureServices(services =>
+                {
+                    services.Configure<HostOptions>(options => options.ShutdownTimeout = TimeSpan.FromSeconds(30));
```

Comment:
I get the argument that if 5 seconds is too short for an arbitrary test timeout, it's too short for a shutdown timeout, but this worries me.

If SignalR is taking a long time to complete it's ongoing requests during server shutdown, I want to know about it and fix it.